<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commanddev</class>
 <widget class="QWidget" name="commanddev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_autoselect">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>自动选取</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_dev">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>931</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_act">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_link">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LINK:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MSG:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mark">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MARK:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_start">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>118</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_end">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>118</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_command">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>下发指令</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_allselect">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_subact">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>对应的订单</string>
   </property>
  </widget>
  <widget class="QLabel" name="order_info">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>订单信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_msg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>193</y>
     <width>831</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_order">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>256</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_direct_send">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>纯指令发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
